src/ResourceManager/Insights/Commands.Insights/LogsCmdletBase.cs

```diff
@@ -105,19 +105,17 @@ protected virtual TimeSpan GetDefaultQueryTimeRange()
         /// <summary>
         /// Sets the max number of records to fetch
         /// </summary>
-        protected virtual void SetMaxEventsIfPresent(string currentQueryFilter, string name, int value)
+        protected virtual void SetMaxEventsIfPresent(string currentQueryFilter, int value)
         {
-            if (value > 0 && value <= 100000)
-            {
-                this.MaxEvents = value;
-            }
+            // If value is not acceptable this forces the use of the default value
+            this.MaxRecords = (value > 0 && value <= 100000) ? value : 0;
```

Comment:
@gucalder just to make sure I understand, does the `MaxEvent` value get set to 1000 somewhere else (by default), or is the description in the change log incorrect?